crates/rolldown/src/module_loader/runtime_module_task.rs

```diff
@@ -120,6 +120,7 @@ impl RuntimeModuleTask {
         },
       },
       css_view: None,
+      asset_view: None,
```

Comment:
Is that possible for a module to have `css` and `asset` module types at the same time?

Reply:
I think no. Though the current design doesn't forbid this kind of combinations, but it shouldn't happen in practice.